srv/salt/ceph/monitoring/prometheus/exporters/files/rbd.sh

```diff
@@ -37,9 +37,13 @@ do
     pools="$(ceph -c $conf osd lspools 2>/dev/null | sed 's/[[:digit:]] \([^,]*\),/\1 /g')"
     for pool in $pools
     do
-        rbd -p $pool --format json -c $conf du 2>/dev/null |
-        jq ".images[] | [.name, .provisioned_size, .used_size] | @csv" |
-        sed -e 's/^"//' -e 's/"$//' -e 's/\\"//g' | # stripe some quotes
-        awk -F , -v cluster=$cluster -v pool=$pool "$awk_cmd"
+        for img in `rbd -p $pool ls`; do
+            if rbd -p $pool info $img | grep "features:.*fast-diff.*" > /dev/null; then
```

Comment:
you can use `grep -q` option and avoid redirecting the stdout to `/dev/null`

`rbd -p $pool info $img | grep -q "features:.*fast-diff.*"`